recipes/podofo/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class OatppSwaggerConan(ConanFile):
+    name = "podofo"
+    license = "GPL-2.0"
+    homepage = "http://podofo.sourceforge.net"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "PoDoFo is a library to work with the PDF file format."
+    topics = ("conan", "PDF", "PoDoFo", "podofo")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+            raise ConanInvalidConfiguration("PoDoFo requires GCC >=5")
+
+    def requirements(self):
+        self.requires("freetype/2.10.4")
+        if str(self.settings.os) in ["Linux", "Macos"]:
+            self.requires("fontconfig/2.13.92")
+        self.requires("libjpeg/9d")
+        self.requires("libunistring/0.9.10")
+        self.requires("libtiff/4.1.0"),
+        self.requires("libidn/1.36")
+        self.requires("openssl/1.1.1h")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("podofo-{0}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["PODOFO_BUILD_LIB_ONLY"] = True
+        if self.options.shared:
+            self._cmake.definitions["PODOFO_BUILD_SHARED"] = True
```

Comment:
I looked further into this.
https://sourceforge.net/p/podofo/code/HEAD/tree/podofo/trunk/CMakeLists.txt#l51

Does this work?
Imho it's better to be explicit.

```suggestion
        self._cmake.definitions["PODOFO_BUILD_SHARED"] = self.options.shared
        self._cmake.definitions["PODOFO_BUILD_STATIC"] = not self.options.shared
```

Reply:
This will work also. If `PODOFO_BUILD_SHARED` is set then static libraries are not build, if not only static are build.
It could be fixed by only one line.